db/sql/cassandra/README.md

```diff
@@ -22,6 +22,18 @@ Also, the client can configure following timeouts:
 - RedialInterval
     - If not zero, gocql attempt to reconnect known DOWN nodes in every ReconnectSleep (ReconnectInterval)
     - Default value is 60s
+- Example
+```go
+    config := &cassandra.Config{
+        Endpoints:      endpoints,
```

Comment:
please put to the example real string with IP & port

Reply:
done.